webapp/src/webapp/webclient/panel.cljs

```diff
@@ -63,20 +69,27 @@
       (reset! timer (js/setTimeout #(save-code-to-localstorage code-string) 1000))
       (reset! script code-string))))
 
-(defn- submit-task [e script selected-connections atom-exec-list-open? metadata]
-  (when (.-preventDefault e) (.preventDefault e))
+(defn- submit-task [e script selected-connections atom-exec-list-open? metadata script-response]
+  (println @script-response)
```

Comment:
feature request to GitHub, let me add gifs from here without having to go to giphy.com